helper/helper.go

```diff
@@ -296,7 +287,13 @@ func DoCleanup() {
 		logVerbose("Encountered error during cleanup: %v", err)
 	}
 
-	for pipeName, _ := range pipesMap {
+	pipeFiles, _ := filepath.Glob(fmt.Sprintf("%s_[0-9]*", *pipeFile))
+	for _, pipeName := range pipeFiles {
+		logVerbose("Signaling pipe %s", pipeName)
```

Comment:
Strictly speaking we can not signal pipe, we send signal to the process which looks like the using pipe one.

Reply:
changed